resources/styles/screen.less

```diff
@@ -127,6 +127,11 @@ html {
 	}
 }
 
+/* Fix to prevent mw-body ul from overriding padding */
+.list-inline, .mw-body ul.list-inline {
```

Comment:
`.list-inline` should not be necessary, it is already covered by https://github.com/twbs/bootstrap/blob/78bdbcd/less/type.less#L172
`.mw-body ul.list-inline` would better be handled in Line 262 by adding
``` less
.list-inline {
  .list-inline
}